packages/viz/src/lib/style/helpers/utils.ts

```diff
@@ -93,7 +93,7 @@ export function hexToRgb(hex: string): RGBAColor {
     ];
   }
 
-  throw new Error();
+  throw new Error(`Error parsing hexadecimal color: ${hex}`);
```

Comment:
We have custom cartoError in case you want to try with it

Reply:
I will take care of it in the validations PRs